### ISSUE TITLE
fog functionality improvement

### ISSUE THREAD

**Issue by bardella** [gvcm/CWAPP#173]
- [ ] evaluate ideal crystal size X fog dept ratio so that front atoms can always be evidenced
- [ ] implement dynamic ratio based on above parameters

**Comment by ThanosSar** [comment]
I don't think this can be done with the current fog features of three.js ( 1. http://threejs.org/docs/#Reference/Scenes/Fog   -   2.http://threejs.org/docs/#Reference/Scenes/FogExp2 ) and the complications that appear in cw because of the objects dynamic change and camera's movement.

I made my research and didn't succed to find a way to do it.
